md/README.md

```diff
@@ -0,0 +1,141 @@
+
+## Extent Detection
+
+The mentions predicted from the previous step is only mention heads. After detecting all mention-heads first, this package also has extent classification for each heads. In the extent classifier training process, each token in the extent to both right and left of a given mention head is formed into pairs to train a binary classifier, indicating whether the token is in the extent or not. To add negative examples, one token to the left and right of the mention extent is added. In the evaluating process, given a mention head, the package forms pairs with tokens to the left and right, until finding a token that is predicted to be not in the extent.
+
+## Results
+
+Head Boundary
+
+| Train\Eval (F1) | ACE  | ERE  |
+|-----------------|------|------|
+| ACE             | 89.6 | 83.8 |
+| ERE             | 81.7 | 86.7 |
+| ACE+ERE         | 88.4 | 86.5 |
+
+Extent boundary accuracy given Head
+
+ACE: 89.45
+
+## Usage
+
+### Using Annotator
+
+```java
+import edu.illinois.cs.cogcomp.annotation.AnnotatorException;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
+import edu.illinois.cs.cogcomp.nlp.utility.TokenizerTextAnnotationBuilder;
+import edu.illinois.cs.cogcomp.annotation.TextAnnotationBuilder;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.nlp.tokenizer.StatefulTokenizer;
+import org.cogcomp.md.MentionAnnotator;
+
+import java.io.IOException;
+import java.util.List;
+
+public class app
+{
+    public static void main( String[] args ) throws IOException, AnnotatorException
+    {
+        String text1 = "Good afternoon, gentlemen. I am a HAL-9000 "
+                + "computer. I was born in Urbana, Il. in 1992";
+
+        String corpus = "2001_ODYSSEY";
+        String textId = "001";
+
+        // Create a TextAnnotation using the LBJ sentence splitter
+        // and tokenizers.
+        TextAnnotationBuilder tab;
+        // don't split on hyphens, as NER models are trained this way
+        boolean splitOnHyphens = false;
+        tab = new TokenizerTextAnnotationBuilder(new StatefulTokenizer(splitOnHyphens));
+
+        TextAnnotation ta = tab.createTextAnnotation(corpus, textId, text1);
+        
+        /*
+         * You can initialize MentionAnnotator with a model parameter or not.
+         * With model parameter like new MentionAnnotator("ERE_TYPE"), you can 
+         * indicate which model the annotator uses. In this case, it is trained on ERE with types.
+         * If you choose to initialize without the parameter like shown below, it is by default
+         * the model trained on ACE without type.
+         * Please refer to the comments in MentionAnnotator.java for more information
+         */
```

Comment:
You mean copy the content of my comments in MentionAnnotator.java to the ReadMe?